frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/obc.scenario.ts

```diff
@@ -0,0 +1,31 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  createNewObjectBucketClaim,
+  deleteObjectBucketClaim,
+  goToObjectBucketClaims,
+  obcStatus,
+} from '../../views/obc.view';
+import { NS, CLAIM_STATUS, STORAGE_CLASS_PATTERNS, DEFAULT_BUCKET_CLASS } from '../../utils/consts';
+
+describe('Test OBC creation with options.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/`);
```

Comment:
Would be better to move this in a constant.

Reply:
Can we leave it the way it is for this PR and introduce this change in a separate PR? We use `${appHost}/` in a few other places as well.